lib/components/molecules/Toggle/Toggle.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable react/prop-types */
+// @flow
+/**
+ *
+ * Toggle
+ *
+ */
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { InputVanilla as Input } from '../../atoms/Input';
+import Label from '../../atoms/Label';
+
+import styles from './Toggle.style';
+
+const Toggle = ({
+  id,
+  name,
+  type,
+  disabled,
+  className,
+  children,
+  labelProps,
+  labelOn,
+  labelOff,
+  labelName,
+  ...otherProps
+}) => {
+  const [toggleOn, handleToggle] = useState(false);
```

Comment:
@vinodloha  Can we use the same naming convention for handling state keys ?
eg: if it is "handle", it should be the same in every component. Small nit picks, but still